src/rebar_prv_cover.erl

```diff
@@ -349,6 +349,8 @@ is_ignored(Dir, File, ExclMods) ->
 
 cover_compile_file(FileName) ->
     case catch(cover:compile_beam(FileName)) of
+        {'EXIT', Reason} ->
+            ?WARN("Cover compilation crashed: ~p", [Reason]);
```

Comment:
yeah, in relx for example if you don't exclude all the templates using `cover_excl_mods` cover compilation will crash, something like:
```
===> Cover compilation crashed: {function_clause,
                                        [{cover,find_main_filename,
                                          [[]],
                                          [{file,"cover.erl"},{line,1434}]},
                                         {cover,transform,2,
                                          [{file,"cover.erl"},{line,1427}]},
                                         {cover,do_compile_beam,3,
                                          [{file,"cover.erl"},{line,1362}]},
                                         {cover,main_process_loop,1,
                                          [{file,"cover.erl"},{line,600}]}]}
```